aws_lambda_powertools/utilities/advanced_parser/envelopes/event_bridge.py

```diff
@@ -0,0 +1,19 @@
+import logging
+from typing import Any, Dict
+
+from pydantic import BaseModel, ValidationError
+
+from aws_lambda_powertools.utilities.advanced_parser.envelopes.base import BaseEnvelope
+from aws_lambda_powertools.utilities.advanced_parser.schemas import EventBridgeSchema
+
+logger = logging.getLogger(__name__)
+
+
+class EventBridgeEnvelope(BaseEnvelope):
```

Comment:
docs: comment or docstring on the return if possible

Reply:
fixed.